tests/common2/unit_tests/test_bgp_route_helper.py

```diff
@@ -0,0 +1,735 @@
+"""
```

Comment:
Filename of unit test files should not start with `test_`. Need to use a different pattern with the general test script files. For example, the `run_test.sh` tool uses pattern `test_*.py` to find all test scripts. Pytest itself also uses this pattern to discover test scripts. Breaking this convention could cause more trouble. It would be better to use different naming pattern for unit test files.

Reply:
Since the unit tests are pytests too I have used the convention recommended by Pytest so that it can find the tests. However in order to avoid from unit tests from being executed by CI or even when running run_tests.sh or pytest directly I have previously committed a change to exclude this by default here - https://github.com/sonic-net/sonic-mgmt/blob/master/tests/pytest.ini#L3. This way it won't interfere with other runs. It needs to be explicitly run and only then it executes.

Reply:
Unfortunately the widely used tool `run_tests.sh` searches  file names matching `tests_*.py`. These unit test files could be included as normal test scripts by that tool.

Another concern is that human could be confused too. At least myself is not aware of the change in `pytest.ini`. If possible, using a different naming pattern for unit test files would be more straight forward for human to understand.